### DISCUSSION TITLE
服务端的环境参数必须在vercel中配置吗？

### DISCUSSION THREAD

**Discussion by btw1024** [walinejs/waline#1960] [General]
- 我试过在waline 的 `.env` 和 `index.js` 文件中配置，提交到GitHub后，vercel自动部署后邮件通知功能无法生效。

**Comment by lizheming** [comment]
是的，Vercel 目前提供了后台修改和命令行修改环境变量的方式，如果有什么意见可以直接 Vercel 提交工单。
